worker/fuzzer/src/RTC/FuzzerRtpPacket.cpp

```diff
@@ -165,8 +165,9 @@ void Fuzzer::RTC::RtpPacket::Fuzz(const uint8_t* data, size_t len)
 	packet->HasExtension(14);
 	packet->GetExtension(14, extenLen);
 	packet->ReadTransportWideCc01(wideSeqNumber);
-	packet->UpdateTransportWideCc01(12345u);
-	packet->SetExtensionLength(14, 2);
+	//packet->UpdateTransportWideCc01(12345u);
```

Comment:
Why are those tests commented? We cannot leave commented tests. Do they no longer work or what?

Reply:
this two line can be replace by setExtensionValue()

Reply:
We should test as much as possible

Reply:
> We should test as much as possible

Extremely agree. This is my first time writing C++, and I have been writing PHP before. So the more tests, the better